Gaijinizer patch/Scripts.txt

```diff
@@ -2829,90 +2829,85 @@ It is already a limit
+...I wonder if this'll work.
+Professor Yui said she'd invented this weird device to be able to
+send messages directly to you, the 'Intervener', so I thought I'd give
+it a try. Apparently the messages come to you in the form of some kind of
+text email.
+
+Well first off, I suppose I should congratulate you. Maybe something
+like, 'Congrats on finishing the game!'. That's right, I know what kind
+of device you've been using to carry out your interventions. That's because
+this isn't the first world that's received very similar intervention.
+There was a world that received similar interventions before. And it's
+always someone from your world who's chosen to intervene.
+
+...Don't worry, I'll keep it a secret from him. ♪
+But I didn't send this message just to congratulate you, I also wanted
+to tell you why you became the 'Intervener'...
+
+Firstly, the fact that you were storing magic while playing in order to
+reach that ending had to be kept under wraps. So that's why your world,
+the world farthest from the Demon Realm was chosen, a world where the
+very notion of 'magic' is nothing but fantasy.
+
+Oh but even though you were storing magic, it was funneled into that
+device you're using, so it won't affect your soul or anything like that.
+Having your soul emptied out just by playing a game would be some kind
+of bad joke.
+
+More importantly is the 'genre' of that device of yours...
+
+Why did you boot up that device? Naturally, everyone has their own
+various reasons for doing what they do, but maybe it's because you have
+some kind of 'preferred situation'?
+
+Even you aren't God, but an ordinary human. You, the intervener, needed the
+'aptitude' to stay with 'him' so he could manage to reach the ending.
+
+Someone who would actually enjoy being raped by a female demon, even perhaps
+a full-on masochist with a monster girl fetish. Someone who's open to non-human
+demons making an appearance.
+
+And above all, someone who can withstand the high difficulty, who can pick
+themselves back up no matter how many times it takes.
+
+If it were just a game, you could weaken the enemies and strengthen your
+characters, but that's not how it works with the 'device'. You can make some
+minor adjustments, but that's about it. What you did change however, was
+what's normally called 'Death's Destiny'. That's not something just any old fool
+can do.
+
+In order to identify those with your 'aptitude', a prototype of the intervention
+device was distributed called the 'Trial Version'. That was quite some time ago,
+do you remember what it contained? Do you remember where and how you intervened
+in that world? It was still an experimental device, so I imagine it was quite
+fragmented.
+
+...What? I'm already out of time? Okay, one last thing!
+This is easy to get confused, but the one who made this device and the one
+who made this game, are likely not the same person. The latter would've
+only thought they were making a niche porn game.
```

Comment:
I'd like to add a little bonus sentence between these two paragraphs, then alter the starting of the second so that it fits:

```
In order to identify those with your 'aptitude', a prototype of the intervention
device was distributed called the 'Trial Version'. That was quite some time ago,
do you remember what it contained? Do you remember where and how you intervened
in that world? It was still an experimental device, so I imagine it was quite
fragmented.



...What? I'm already out of time? Okay, one last thing!
This is easy to get confused, but the one who made this device and the one
who made this game, are likely not the same person. The latter would've
only thought they were making a niche porn game.
```